packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/UI/UI.md

```diff
@@ -313,3 +316,11 @@ Actions can be added to conversations with `menuItems`. Optionally, you can also
 ```js file="./ChatbotHeaderDrawerWithActions.tsx"
 
 ```
+
+### Modal
+
+Based on the [PatternFly modal](/components/modal), this modal adapts to the chatbot display mode and accepts components typically used in a modal. It is primarily used and tested in the context of the [attachment modals](/patternfly-ai/chatbot/messages#attachment-preview), but you can customize this modal to adapt it to other use cases as needed.
```

Comment:
Is it true and/or worth noting that the modal will overlay the chatbot depending on the display mode of the chatbot? and should that something we can demonstrate in the example?

Reply:
I can add it to the example. What do you think @edonehoo? I'm fine with adding that; maybe this?

Based on the [PatternFly modal](/components/modal), this modal adapts to the chatbot display mode and accepts components typically used in a modal. It is primarily used and tested in the context of the [attachment modals](/patternfly-ai/chatbot/messages#attachment-preview), but you can customize this modal to adapt it to other use cases as needed. The modal will overlay the chatbot in default and docked modes, and will behave more like a traditional PatternFly modal in fullscreen and embedded modes.

Reply:
I think that sounds great!

Reply:
Thanks! Just updated both the text and example; let me know if you like this any better.